rest_framework_swagger/docgenerator.py

```diff
@@ -20,6 +20,9 @@ class DocumentationGenerator(object):
     # Response classes defined in docstrings
     explicit_response_types = dict()
 
+    def __init__(self, for_user):
```

Comment:
It's actually only done once, the first time the class body is processed by the python interpreter. You may recall this as the reason why you can't put an empty list literal as the default for a keyword argument. :white_check_mark: